go-controller/pkg/cluster/gateway_init.go

```diff
@@ -126,12 +150,20 @@ func CleanupClusterNode(name string, kube *kube.Kube) error {
 
 	switch config.Gateway.Mode {
 	case config.GatewayModeLocal:
-		err = cleanupLocalnetGateway()
+		err = cleanupLocalnetGateway(util.PhysicalNetworkName)
 	case config.GatewayModeSpare:
 		nodeName := strings.ToLower(node.Name)
 		err = cleanupSpareGateway(config.Gateway.Interface, nodeName)
 	case config.GatewayModeShared:
-		err = cleanupSharedGateway()
+		err1 := cleanupLocalnetGateway(util.LocalNetworkName)
+		err2 := cleanupSharedGateway()
+		if err1 == nil {
+			err = err2
+		} else if err2 == nil {
+			err = err1
+		} else {
+			err = fmt.Errorf("Combined errors: %v, %v", err1, err2)
+		}
```

Comment:
Since this function does cleaning on best-effort-bassis, can we just log the error at every stage..

```
case config.GatewayModeShared:
    err  = cleanupLocalnetGateway(util.LocalNetworkName)
    if err != nil {
         // log error
     }
     err =  cleanupSharedGateway()
```